project.json

```diff
@@ -82,6 +85,16 @@
     "lint-pnpm-lock": {
       "cache": true,
       "inputs": ["{projectRoot}/pnpm-lock.yaml"]
+    },
+    "prepush": {
+      "dependsOn": [
+        "format-native",
+        "lint-native",
+        "documentation",
+        "check-commit",
+        "check-format:quick",
+        "check-lock-files"
```

Comment:
I don't think this works quite the way you want - this only adds a dep to the current projects targets, which doesn't exist for format-native / format-lint. Those 2 need to be qualified with a project name.